sdks/python/.pylintrc

```diff
@@ -178,6 +179,7 @@ indent-after-paren=4
 ignore-long-lines=(?x)
   (^\s*(import|from)\s
    |^\s*(\#\ )?<?(https?|ftp):\/\/[^\s\/$.?#].[^\s]*>?$
+   |# type:
```

Comment:
This line disabled line-too-long errors across the board: https://issues.apache.org/jira/browse/BEAM-9058.